src/core/utils/geometryutils.cpp

```diff
@@ -102,6 +102,11 @@ GeometryUtils::GeometryOperationResult GeometryUtils::addRingFromRubberband( Qgs
 {
   QgsPointSequence ring = rubberBandModel->pointSequence( layer->crs(), layer->wkbType(), true );
 
+  if ( ring.size() < 3 )
```

Comment:
Separate PR?

Reply:
@m-kuhn , this is relevant to this PR, it fixes the only falling test we had upgrading to qt6